src/Employee.java

```diff
@@ -0,0 +1,58 @@
+package Base;
+
+import java.util.Objects;
+
+public class Employee {
+    private String name;
+    private String department;
```

Comment:
Тут нужно использовать тип int для отдела. 
```suggestion
    private int department;
```

Reply:
Соответственно, в конструкторе и в геттере и сеттере тоже нужно поменять тип.